unit/eq_test.c

```diff
@@ -540,6 +580,7 @@ struct test_entry test_array[] = {
 	TEST_ENTRY(eq_wait_fd_sread, "Test EQ sread"),
 	TEST_ENTRY(eq_wait_read_peek, "Test EQ read with FI_PEEK"),
 	TEST_ENTRY(eq_wait_sread_peek, "Test EQ sread with FI_PEEK"),
+	TEST_ENTRY(eq_readerr_ret_eagain, "Test eq_readerr_ret_eagain"),
```

Comment:
nit: Please describe the test in simpler terms, not the name of the function being called.